client/src/views/Dashboard/Summary/index.js

```diff
@@ -682,6 +699,10 @@ export default class Summary extends PureComponent {
             mostVulnerableChildrenVulnerabilityMarker,
         );
 
+        const healthNutritionWithName = this.getDonutDataWithName(healthNutrition);
+        const educationInvolvementWithName = this.getDonutDataWithName(educationInvolvement);
+        const childMonitoringVizDataWithName = this.getDonutDataWithName(childMonitoringVizData);
```

Comment:
Calling the same memoized function with three different values consecutively results in cache miss every time.